dev/check-license

```diff
@@ -44,7 +44,7 @@ acquire_rat_jar () {
     # We failed to download
     rm "$JAR"
     printf "Our attempt to download rat locally to ${JAR} failed. Please install rat manually.\n"
-    exit -1
+    exit 255
```

Comment:
Can we get an explanation for a few of the changes that occur frequently? like is this equivalent to returning -1?

Reply:
SC2242 (error): Can only exit with status 0-255. Other data should be written to stdout/stderr.

Reply:
BTW，`shellcheck` support exclude check rules, thus if we support `exit -1`, we can excude SC2242 in GA.